Source/Bogus.Tests/GitHubIssues/Issue260.cs

```diff
@@ -30,11 +33,12 @@ public void fast_algo3_test()
 
          // last 2^14 = 16384, for last 4 digits of SSN
          var c = (x >> 7) & 0x3FFF;
+         if( c >= 10000 ) c -= 10000;
```

Comment:
Bug in test: `& 16383` does _not_ guarantee a 4 digit value.

Reply:
Actually, the whole algorithm for `fast_algo3_test()` for variables `a` and `b` are broken:
![image](https://user-images.githubusercontent.com/478118/84577548-18aba300-ad72-11ea-9f3c-e37e02cb250c.png)
Good thing **ssn algo3** is not used in production.

IIRC, when I was testing issue 260, this was more of a test if I could squeeze more clock cycles out of the current SSN generator. It turned out, not much was to be gained going this far and ultimately stuck with the easier to understand implementation. :smile:

Reply:
Ahh :-) So, for this PR, should I enhance `fast_algo3_test` to detect the errors and fix them, or just tear the test out entirely?

Reply:
I mean, I guess the test isn't really testing a subject, all the logic is _in the test_, so if it's not being used ... tear it out?? It needs _some_ change because with the change to `Random.Number`, it no longer passes.

Reply:
Right on. For now, just making the test pass with the assertion should be enough. We can remove the bad algo separately from the PR; we just need this particular test to pass CI.

Currently benchmarking your new `Random.Number` with some alternatives. I'll post shortly.

Reply:
Okay, then we're probably good then, right? Even if the algorithm is broken, the _test_ doesn't detect it presently, it only hits valid results. The test is green on my end.

Reply:
Yea, we're good. :+1: